src/Components/FilterButton.jsx

```diff
@@ -0,0 +1,19 @@
+
+import PropTypes from "prop-types";
+function FilterButton({ label, color, onClick, hovercolor }) {
```

Comment:
what is the use of this file?

Reply:
ma'am this file is used to filter tasks for the three buttons(All,completed and inCompleted buttons).

Reply:
use the common button , u dont need to create multiple common buttons